src/content/components/PriorityGuide/PriorityGuide.scss

```diff
@@ -1,6 +1,5 @@
 .container {
   padding: 20px;
-  background: #f9f9fa;
   width: 250px;
```

Comment:
You're also going to want to transition the width so this shrinks along with the inner content